---
layout: post
title: "Single-cell molecular connectomics of intracortically-projecting neurons"
byline: Klingler et al
doi: "10.1101/378760"
tags:
    - neuroscience
    - cortex
    - mouse
    - transcriptome
    - S1
    - M1
    - A1
    - V1
    - thalamus
    - MAPseq
    - RNA
    - barcode
    - connectome
summary: Combining MAPseq with RNA sequencing, ConnectID enables researchers to overlay both transcriptome as well as and connectome information in cortical tissue.
---

Classical single-cell connectomics has required huge anatomical datasets, and because the imaging takes place ex-vivo, generally, these datasets lack functional or physiological information. This means that genetic and cell-type information is lost.

MAPseq is a procedure in which researchers genetically barcode neurons, wait for the neuron to transport those DNA barcodes to the neural "destinations" (namely, synapses), and then perform DNA-sequencing on the neural tissue in order to determine breadth and spread of those neurons. Because of the enormous combinatorial variety of possible genetic barcodes (see also: antibody random assortment for receptor diversity), it is generally possible to distinguish individual neurons (even in the presence of noisy measurements).

This protocol, dubbed ConnectID, combines MAPseq with RNA sequencing in order to determine the native gene expression of the neuron. In this way, it is possible to collect both the projection map of the neurons, as well as core information about the neurons, such as neuron type, associated neurotransmitters...perhaps even cell lineage.

Using ConnectID, (which I think is also the name of a cloud authentication provider, right?) the researchers were able to establish both a projection map from S1 to several other brain areas, and explain the distributions of neuron-type diversity in each of those areas.
